drivers/usb/usbuhci_new/usbuhci.c

```diff
@@ -772,6 +773,8 @@ UhciStopController(IN PVOID uhciExtension,
             DbgBreakPoint();
             break;
         }
+
+        Command.AsUSHORT = READ_PORT_USHORT(CommandReg);
     }
```

Comment:
I had a slightly different variant:
-    while (((UHCI_USB_COMMAND)READ_PORT_USHORT(CommandReg)).HcReset == 1)
+    while (Command.AsUSHORT = READ_PORT_USHORT(CommandReg),
+           Command.HcReset == 1)

Reply:
Feel free to overwrite mine, I just wanted to have the build green.